components/playground/SelectedPolicy.js

```diff
@@ -14,61 +14,33 @@
  * limitations under the License.
  */
 
-import React, { useState } from "react";
+import React from "react";
 import PropTypes from "prop-types";
-import Button from "components/Button";
 import styles from "styles/modules/Playground.module.scss";
-import typeStyles from "styles/modules/Typography.module.scss";
 import Code from "components/Code";
-import LabelWithValue from "components/LabelWithValue";
+import textStyles from "styles/modules/Typography.module.scss";
 
 const SelectedPolicy = (props) => {
-  const { policy, clearPolicy } = props;
-
-  const [showDetails, setShowDetails] = useState(false);
-  const toggleDetails = () => setShowDetails(!showDetails);
+  const { policy } = props;
 
   return (
-    <div className={styles.selectionContainer}>
-      <h2 className={styles.selectionTitle}>
-        Selected Policy
-        <span className={styles.selectionName}>{policy.name}</span>
-      </h2>
-      <div className={styles.selectionButtonContainer}>
-        <Button
-          buttonType={"text"}
-          label={showDetails ? "Hide Policy Details" : "Show Policy Details"}
-          onClick={toggleDetails}
-        />
-        <Button
-          buttonType={"textDestructive"}
-          label={"Clear Policy"}
-          onClick={clearPolicy}
+    <>
+      <p className={textStyles.label}>Rego Policy Code</p>
+      {policy ? (
+        <Code
+          code={policy.regoContent}
+          language={"rego"}
+          className={styles.codeContent}
+          data-testId={"regoPolicyCode"}
```

Comment:
This was emitting a warning in the console:
>Warning: React does not recognize the `data-testId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `data-testid` instead. If you accidentally passed it from a parent component, remove it from the DOM element.


```suggestion
          data-testid={"regoPolicyCode"}
```